src/api/app/components/button_component.rb

```diff
@@ -0,0 +1,17 @@
+class ButtonComponent < ApplicationComponent
+  renders_one :icon
+  renders_one :text
+  renders_one :modal
+
+  def initialize(type: nil, css_custom: '', button_data: {})
+    super
+
+    @type = type
+    @css_custom = css_custom
+    @button_data = button_data
+  end
+
+  def css_style
+    (@type ? "btn-#{@type} " : 'btn-light border-1 border-gray-300 ') << @css_custom
```

Comment:
This is how `.btn-secondary` looks like:
![image](https://github.com/openSUSE/open-build-service/assets/7080830/441b017e-9b22-4bc8-9eea-62d04b396647)

and this is how `.btn-info` looks like:
![image](https://github.com/openSUSE/open-build-service/assets/7080830/4892bc29-cc41-4884-9b2b-5ac31b5119e1)

I think there is not so much of a difference so that's why I don't like to use the secondary option.
We should change https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/stylesheets/webui/bootstrap_variables/colors.scss#L18-L22 but I it's too invasive and out of scope for this PR.

What do you think?

Reply:
Let's do it, I don't think it's gonna hurt too much.